### DISCUSSION TITLE
Contribution and maintenance

### DISCUSSION THREAD

**Discussion by Moerliy** [jackMort/ChatGPT.nvim#342] [General]
I love using the plugin and a bunch of people do so too, but I have come to notice that maintenance is sometimes a bit lackluster.
I see a bunch of issues that would improve the overall experience and even propper pull requests.
While some people open issues others even take their time to fix and implement things themself. I get the feeling that I'm not the only one that wants to see this plugin thrive.

I understand that maintaining a well-used repository is time-consuming and people have other things to do than keeping tabs on their side project that coincidentally became pretty popular. But it kinda makes me sad seeing the process on a repository stall even if the community contribution is high.

If the maintenance is too much I would suggest asking the community for help. I think there would be some people who would love to give a helping hand.

I would like to have some extra opinions on this.

**Comment by rameshsanth** [comment]
agree. this one seems to be going down the same path. PR's keep piling up
